### ISSUE TITLE
Parsing anonymous nested structs fails

### ISSUE THREAD

**Issue by nics** [g4s8/envdoc#5]
Parsing this config struct panics on the nested struct:

```go
type Config struct {
	Repo struct {
		Conn  string  `env:"CONN,notEmpty"`
	} `envPrefix:"REPO_"`
}

```

Error message:

```
main.(*inspector).parseField(0xc0000ca400, 0xc0000f2380)
        /Users/xxx/go/pkg/mod/github.com/g4s8/envdoc@v0.0.10/inspector.go:207 +0x678
main.(*inspector).Visit(0xc0000ca400, {0x1236658?, 0xc0000f21c0?})
        /Users/xxx/go/pkg/mod/github.com/g4s8/envdoc@v0.0.10/inspector.go:136 +0x2e5
go/ast.Walk({0x1236018?, 0xc0000ca400?}, {0x1236658?, 0xc0000f21c0?})
        /usr/local/Cellar/go/1.21.6/libexec/src/go/ast/walk.go:51 +0x5c
go/ast.Walk({0x1236018?, 0xc0000ca400?}, {0x1236ec8?, 0xc0000f2400?})
        /usr/local/Cellar/go/1.21.6/libexec/src/go/ast/walk.go:344 +0x1b85
go/ast.walkDeclList(...)
        /usr/local/Cellar/go/1.21.6/libexec/src/go/ast/walk.go:38
go/ast.Walk({0x1236018?, 0xc0000ca400?}, {0x1236608?, 0xc0000ba8c0?})
        /usr/local/Cellar/go/1.21.6/libexec/src/go/ast/walk.go:366 +0x1b05
main.(*inspector).inspect(0xc0000ca400, {0x1236608?, 0xc0000ba8c0?})
        /Users/xxx/go/pkg/mod/github.com/g4s8/envdoc@v0.0.10/inspector.go:66 +0x5b
main.(*inspector).inspectFile(0xc0000ca400, {0xc000012107, 0x9})
        /Users/xxx/go/pkg/mod/github.com/g4s8/envdoc@v0.0.10/inspector.go:61 +0x128
main.(*generator).generate(0xc0000f0600, {0x1235fd8, 0xc000090400})
        /Users/xxx/go/pkg/mod/github.com/g4s8/envdoc@v0.0.10/generator.go:91 +0xc5
main.run(0xc0000e7e60)
        /Users/xxx/go/pkg/mod/github.com/g4s8/envdoc@v0.0.10/main.go:116 +0x618
main.main()
        /Users/xxx/go/pkg/mod/github.com/g4s8/envdoc@v0.0.10/main.go:64 +0x112
```

Thanks for a handy tool!

**Comment by g4s8** [comment]
Thanks for reporting! I'll look into this problem

**Comment by g4s8** [comment]
The fix should be available in [v0.1.1](https://github.com/g4s8/envdoc/releases/tag/v0.1.1) version.

**Comment by nics** [comment]
Hi @g4s8 thanks it works now. Note that I get a segfault if there is no description comment.

So this works:
```
type Config struct {
       // Repository configuration
       Repo struct {
		Conn  string  `env:"CONN,notEmpty"`
	} `envPrefix:"REPO_"`
}
```
But this segfaults:
```
type Config struct {
	Repo struct {
		Conn  string  `env:"CONN,notEmpty"`
	} `envPrefix:"REPO_"`
}
```

**Comment by g4s8** [comment]
Sorry, my bad. The panic was fixed in [v0.1.2](https://github.com/g4s8/envdoc/releases/tag/v0.1.2).
